packages/generator/src/imports.ts

```diff
@@ -59,15 +62,17 @@ export function corePropertyTypeImportNames(
 export function corePropertyFieldTypeImportNames(
   properties: VdmProperty[]
 ): string[] {
-  return unique(
-    properties.filter(prop => !prop.isComplex).map(prop => prop.fieldType)
-  );
+  return unique([
+    ...properties.filter(prop => !prop.isComplex).map(prop => prop.fieldType),
+    ...(properties.some(prop => prop.isCollection) ? ['CollectionField'] : [])
```

Comment:
Minor: Isn`t there a more elegant way to add the 'CollectionField' to the list? Wrapping it inside an [] and then ... it feels kind of brute force.

Reply:
The only way I can think of would be:
```
const x = unique(properties.filter(prop => !prop.isComplex).map(prop => prop.fieldType));
return properties.some(prop => prop.isCollection) ? [...x, 'CollectionField'] : x;
```
I guess then it is a matter of taste. I think I like my approach above more ;)

Reply:
aggreed.